src/index.html

```diff
@@ -5,9 +5,44 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body data-qa="card">
+    <section class="card">
+      <div class="product_image">
+        <img src="images/imac.jpeg" alt="iMak">
+      </div>
+
+      <h2 class="product_title">
+        APPLE A1419 iMac 27" Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product_code">Product code: 195434</p>
+
+      <div class="stars">
+        <section class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </section>
+        <p class="stars__title">Reviews: 5</p>
+      </div>
+      <div class="price">
+        <p class="price__title">Price:</p>
+        <p class="price__number">$2,199</p>
+      </div>
+      <a href="#"
```

Comment:
Add the blank line between the blocks for better readability
```suggestion
      </div>
      
      <div class="price">
        <p class="price__title">Price:</p>
        <p class="price__number">$2,199</p>
      </div>
      
      <a href="#"
```